app/controllers/api/v1/materials_controller.rb

```diff
@@ -307,8 +347,288 @@ def assign_to_class
     render json: {:message => message}, :status => status
   end
 
+
+  #
+  # Get the standards associated with a material
+  #
+  # params[:material_type]  The material type
+  # params[:material_id]    The material ID
+  #
+  def get_materials_standards
+
+    uri     = params[:identifier]
+    type    = params[:material_type]
+    id      = params[:material_id]
+
+    statements = 
+        StandardStatement.find_all_by_material_type_and_material_id(
+            type,
+            id,
+            :order => "uri ASC")
+
+    ret = []
+
+    statements.each do |statement|
+   
+        ret.push( {
+            uri:                statement.uri,
+            description:        statement.description,
+            statement_label:    statement.statement_label,
+            statement_notation: statement.statement_notation,
+            doc:                statement.doc,
+            is_applied:         true
+        })
+
+    end
+
+    render json: {:statements => ret}, :status => 200
+ 
+  end
+
+
+  #
+  # Add a standard to a material
+  #
+  # params[:identifier]     The standard statement identifier (URI)
+  # params[:material_type]  The material type
+  # params[:material_id]    The material ID
+  #
+  def add_materials_standard
+
+    key     = ENV['ASN_API_KEY']
```

Comment:
It might be nice for deployment debugging to throw a 500 error here if there is no value for `ASN_API_KEY`.  The same goes for the other endpoints.